src/modules/Security.rb

```diff
@@ -858,6 +830,48 @@ def Overview
     publish :function => :Export, :type => "map ()"
     publish :function => :Summary, :type => "list ()"
     publish :function => :Overview, :type => "list ()"
+
+    protected
+
+    # Sets @missing_mandatory_services honoring the systemd aliases
+    def read_missing_mandatory_services
+      log.info("Checking mandatory services")
+
+      @missing_mandatory_services = @mandatory_services.select do |services|
+        enabled = services.any? { |service| Service.enabled?(service) }
+        log.info("Mandatory services #{services} are enabled: #{enabled}")
+        !enabled
+      end
+
+      log.info("Missing mandatory services: #{@missing_mandatory_services}")
+    end
+
+    # Sets @extra_services honoring the systemd aliases
+    def read_extra_services
+      log.info("Searching for extra services")
+
+      enabled_services = SystemdService.all(names: "Names").select(&:enabled?)
+      mandatory = @mandatory_services.flatten
+      @extra_services = enabled_services.select do |service|
+        # A service is 'extra' if it's neither mandatory or optional
+        extra = !mandatory.include?(service.name) && !@optional_services.include?(service.name)
+        # If we failed to find it in the lists by name, try by alias
+        if extra
+          names = service.properties.names
+          if names
```

Comment:
NP: This looks pretty complicated, CodeClimate would definitely complain here, maybe split this into more methods...